client/my-sites/themes/thanks-modal.jsx

```diff
@@ -201,20 +201,22 @@ const ThanksModal = React.createClass( {
 
 export default connect(
 	( state, { site } ) => {
+		let siteIdOrWpcom;
```

Comment:
The thanks modal is a bit different from other parts (like e.g. the theme sheet) in that it shouldn't fall back to 'wpcom'. The reason is that by definition, the thanks modal will only be shown when the user has activated a theme for a given site -- you can't activate a theme e.g. for a `null`/`undefined` site, or for all of `wpcom` :smile:

So the `! site` check here only covers the case when the `site` prop, e.g. when coming from multi-site mode via the site selector modal, just hasn't been updated yet -- there can be a slight delay. So we need to guard the selectors against this case (and have them return fallback values) -- but those don't have anything to do with 'wpcom', so we can't fall back to that value.